documentation/governance.md

```diff
@@ -0,0 +1,148 @@
+# Governance and Revision Control 
+```eval_rst
+
+.. admonition:: Download
+
+  You can also download this document in PDF format :download:`here <../assets/governance_and_review_process.pdf>`.
```

Comment:
Not sure this is very good from an SSOT perspective.

Reply:
@morchickit thoughts? Is it necessary to have a PDF document for download?

Reply:
I see Ben's point. We can remove the PDF then.

Reply:
Removed now.